modules/inbucket/inbucket.go

```diff
@@ -57,8 +57,12 @@ func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomize
 func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustomizer) (*InbucketContainer, error) {
 	req := testcontainers.ContainerRequest{
 		Image:        img,
-		ExposedPorts: []string{"2500/tcp", "9000/tcp"},
-		WaitingFor:   wait.ForLog("SMTP listening on tcp4"),
+		ExposedPorts: []string{"2500/tcp", "9000/tcp", "1100/tcp"},
```

Comment:
Why do we need to expose the POP3 port here? If it's needed, then I'd label this PR as `feature` instead.